apps/beatfolio/.env.example

```
@@ -6,7 +6,7 @@ BASIC_AUTH_USERNAME=
 BASIC_AUTH_PASSWORD= 
 
 AUTH0_SECRET=''
-APP_BASE_URL=''
+APP_BASE_URL='' # 未設定時は VERCEL_URL → localhost:3000 の順で自動決定
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The APP_BASE_URL key should go before the AUTH0_SECRET key

(UnorderedKey)